### DISCUSSION TITLE
Messages only seen by a certain user

### DISCUSSION THREAD

**Discussion by xSavgs** [Rapptz/discord.py#6355] [Ideas] [answered]
Could you make it so that only a specific user could see a message that a bot sends

It's possible discord can do this, hence this:

![image](https://user-images.githubusercontent.com/53839710/106077639-cb662100-6109-11eb-9c7c-ba30cfe57a80.jpeg)

Is it possible to build this into the discord.py library?

It could maybe work like, 

```
await ctx.send("message", author=ctx.author)
```

**Comment by Zomatree** [comment]
The Discord API does not have a way to do this for any message, however, this is possible with the new slash commands, but discord.py does not support them. see #6149 for reasons why.

**Comment by Rapptz** [comment, marked as answer]
Clyde is a client-sided bot. It is not an actual bot.

That aside, this feature is called ephemeral messages and is only supported by interactions. Its support is incomplete Discord side since it does not allow embeds or attachments currently. As a result, they're not in the library.

**Comment by nevertoday** [comment]
how to export all message logs include those message: only you can see ? 
api can't ?
